pytorch_pfn_extras/handler.py

```diff
@@ -584,7 +584,8 @@ def train_step(self, models, optimizers, batch_idx, batch):
         to_back_outs = outs
         if self._grad_scaler is not None:
             to_back_outs = self._normalize_outputs(outs)
-            assert len(outs) == 1, "loss scaling with multiple outputs is not supported"
+            if not isinstance(outs, torch.Tensor):
+                assert len(outs) == 1, "loss scaling with multiple outputs is not supported"
```

Comment:
```suggestion
             assert len(to_back_outs) == 1, "loss scaling with multiple outputs is not supported"
```

I think that `_normalize_outputs` will create a dict with the tensor, so this should be enough